docs/conf.py

```diff
@@ -273,25 +277,70 @@
 
 suppress_warnings = ['image.nonlocal_uri']
 
-# build the dynamic files in _include
+
+
 def build_includes():
     """Creates rst files in the _include directory using the python scripts
     there.
 
     This will ignore any files in the _include directory that start with ``_``.
     """
-    print("Running scripts in _include:")
+    logging.info("Running scripts in _include:")
     cwd = os.getcwd()
     os.chdir('_include')
     pyfiles = glob.glob('*.py') + glob.glob('*.sh')
+    run_args = []
     for fn in pyfiles:
         if not fn.startswith('_'):
-            print(' {}'.format(fn))
             if fn.endswith('.py'):
-                subprocess.check_output(['python', fn])
+                exe = 'python'
             elif fn.endswith('.sh'):
-                subprocess.check_output(['bash', fn])
-            print('DONE with {}'.format(fn))
+                exe = 'bash'
+             
+            args = [exe, fn]
+            run_args.append(args)
+    
+    run_num = 2
```

Comment:
I guess this is 2 because of the CI runners ... But building docs locally, 2 might not be the best choice. Not sure how to make this choice at a higher level??

Reply:
Yeah, that's tricky. There's also the problem that memory useage may or may not be conducive to doing something simple like setting it to the number of available threads.